lib/busbar_cli/commands/logs.rb

```diff
@@ -3,43 +3,34 @@ module Logs
     extend ActiveSupport::Concern
 
     included do
-      desc 'logs APPLICATION_CONTAINER ENV [COMPONENT_TYPE]',
-        'Fetch the logs from a application container. You can find the container for your application through the "busbar containers" command.'
+      desc 'logs APP ENV [COMPONENT]',
+        'Fetch the logs from a application.'
```

Comment:
What do you think of?
```suggestion
        'Stream the application logs.'
```

Reasons:
* Fetch implies that the operation happens once, whereas in reality it's using a `tail`-like mechanism
* "a application" was a typo; I figured it would read better by shuffling the words around